organizing_hub/tasks.py

```diff
@@ -397,7 +401,29 @@ def sync_contact_list_with_bsd_constituents(
 """Tasks"""
 
 
-@shared_task
+class BaseTask(Task):
```

Comment:
This looks like same code from above, can we avoid copying/pasting code and put in central location?